common/src/main/resources/tachyon-default.properties

```diff
@@ -122,10 +122,9 @@ tachyon.user.block.size.bytes.default=512MB
 tachyon.user.failed.space.request.limits=3
 tachyon.user.file.buffer.bytes=1MB
 tachyon.user.file.master.client.threads=10
-tachyon.user.file.tachyonstoragetype.default=PROMOTE
-tachyon.user.file.understoragetype.default=NO_PERSIST
 tachyon.user.file.waitcompleted.poll.ms=1000
-#tachyon.user.file.writetype.default=MUST_CACHE
+tachyon.user.file.writetype.default=MUST_CACHE
+tachyon.user.file.readtype.default=CACHE_PROMOTE
```

Comment:
please update web doc on this "new" property.
also maybe remove "tachyon.user.file.tachyonstoragetype.default" and "tachyon.user.file.understoragetype.default"?